app/views/static/cloud_volume_backup/volume_select.html.haml

```diff
@@ -11,11 +11,11 @@
       %label.col-md-2.control-label
         = _('Volume')
       .col-md-8
-        = select_tag('volume_id',
-                '',
+        %select{:id          => 'volume_id',
+                :name        => 'volume_id',
                 'ng-model'   => 'vm.cloudVolumeBackupModel.volume',
                 'ng-options' => 'volume.name for volume in vm.volume_choices track by volume.id',
                 :required    => '',
-                'miq-select' => true)
+                'miq-select' => '{ noneSelectedText: "&lt;#{_('Choose a volume')}&gt;" }'}
```

Comment:
bad gettext... ruby interpolation doesn't work in single-quoted strings

Maybe something like..

```haml
- text = "&lt;#{_('Choose a volume')}&gt;"
...
   'miq-select' => "{noneSelectedText: '#{j_str text}'}",
```